### ISSUE TITLE
Movement bug/exploit in arena

### ISSUE THREAD

**Issue by Goatnip** [dalaranwow/dalaran-wow#5118]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Hey, so I was just starting to queue up some arenas for the day with my partner (me being Goatmeister, a warlock and him Incineration, a mage if it's relevant) and pretty quickly we got matched with a hunter + retri (Velaa and Bubbleberry) pally  combo, it starts out well enough until I get Hammer of Justice on me, which i use trinket for, then i get chain scattered into freezing trap. I sit out the entire duration of both of those but when it expires I find that I can't move. I'm stuck in place without ANY debuffs. I used my circle of summoning to try to fix the problem but that didn't help, neither did escape artist.
Thinking this was just a freak accident we queued again after being defeated because of this. However, the same thing happened, i got hit by hammer of justice, which i quickly trinketed, then hit by scatter shot, and again I was stuck. Once again I used my circle but was stuck there as well. I could no nothing as my partner was killed and I was stuck behind a wall.
2.  I should be able to move when I have no debuffs on my character. Right now I don't have any proof of this happening as this would require us to queue up against them again and for them to use that same exploit again.
3. After some asking around I heard there was a bug with hunters where after using scatter shot and the pet stun ability simultaneously will cause this to happen. So supposedly that is a way to do recreate it.

EDIT: After queuing up against a rogue/priest composition the same thing happened after cheap + kidney shot. After restarting the client however the bug was gone. While this step could've been taken sooner it'd be nice to not have to lose rating because of something like this occurring so if something could be done about it that'd be appreciated.
**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by KerDW** [comment]
I was the guy playing the hunter, Velaa, I believe this was your character being buggy rather than us exploiting anything. I haven't ever exploited or encountered this bug.

"After some asking around I heard there was a bug with hunters where after using scatter shot and the pet stun ability simultaneously will cause this to happen. So supposedly that is a way to do recreate it."

Never heard of that or abused it either, I certainly didn't do that against you. Besides, you don't just post bugs and even less accuse anyone without solid proof. 

Cheers.

**Comment by kysqt** [comment]
It's a common bug and occurrence, after scattershot into ravanger pet stun or some other combination of abilities your character cannot move in the direction you were trying to move before hand, to fix it you need to strafe (not keyboard turn) in a different direction, after a second it will fix your problem.

**Comment by KerDW** [comment]
Hm, wasn't aware of that, it could be related to the disorient/fear "fix" that was applied not long ago.

**Comment by TeridanDW** [comment]
Thank you everyone,
I can't reproduce it on PTR, would be great if someone write "Steps to reproduce the problem:" or even better a video from Algalon.
Thank you.

**Comment by Rushor** [comment]
Closed until proof or "Steps to reproduce the problem:" were added. Thanks
